taichi/ir/expr.cpp

```diff
@@ -161,7 +161,7 @@ void Expr::operator/=(const Expr &o) {
 }
 
 Expr load_if_ptr(const Expr &ptr) {
-  if (ptr.is<GlobalPtrExpression>()) {
+  if (ptr.is<GlobalPtrExpression>() or ptr.is<GlobalTensorElementExpression>()) {
```

Comment:
nit:
```suggestion
  if (ptr.is<GlobalPtrExpression>() || ptr.is<GlobalTensorElementExpression>()) {
```
`or` is not supported by some compilers...